packages/angular/cli/commands/config-impl.ts

```diff
@@ -32,6 +32,9 @@ const validCliPaths = new Map([
   ['cli.warnings.versionMismatch', 'boolean'],
   ['cli.defaultCollection', 'string'],
   ['cli.packageManager', 'string'],
+  ['cli.analytics', 'string'],
```

Comment:
Why are these being used?

Reply:
`ng config cli.analytics=abc`. I think the whole config command is a bit weird in its type checking, but it's not the right time to refactor that one yet.